### PR TITLE
Removed tweak for ie < 9

### PR DESCRIPTION
**Pull request by StefanSpieker**
Just a small method fo taking care of older IE versions, I guess it can be removed.

### Proposed changelog entries

* N/A

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by res0nance — APPROVED** [review]
Seems reasonable

**Review by oleg-nenashev — APPROVED** [review]
We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

File: war/src/main/webapp/scripts/msie.js
Comment (timja) [inline war/src/main/webapp/scripts/msie.js:44]: shouldn't respond.js also be removed?

Comment (res0nance) [reply war/src/main/webapp/scripts/msie.js:44]: Likely, a quick search shows that this is the only place it was used

Comment (StefanSpieker) [reply war/src/main/webapp/scripts/msie.js:44]: Yes you are right. Should I remove it within this PR or create a new one?

**Comment by timja** [comment]
@oleg-nenashev we should remove the polyfill that this is adding before merging

File: war/src/main/webapp/scripts/msie.js
Comment (timja) [reply war/src/main/webapp/scripts/msie.js:44]: in this PR, it's a polyfill for this functionality

Comment (StefanSpieker) [reply war/src/main/webapp/scripts/msie.js:44]: done.

**Review by timja — APPROVED** [review]

**Pull request merged**
